.depend

```diff
@@ -6073,6 +6073,3 @@ driver/compdynlink.cmo : \
     driver/compdynlink_common.cmi \
     bytecomp/cmo_format.cmi \
     driver/compdynlink.cmi
```

Comment:
These dependencies were repeated.

Reply:
Isn't this file auto-generated? Why is there a repetition here?

Reply:
Ah, I suppose the removal diff is not manual, but a result of the Makefile changes you did below.

Reply:
Exactly.